Prac_05/word_occurrences.py

```diff
@@ -0,0 +1,15 @@
+NUM_OF_WORDS = {}
+KEY_LIST = []
+
+text = input("Text: ")
+KEY_LIST = text.split()
+KEY_LIST.sort()
+NUM_OF_WORDS = dict.fromkeys(KEY_LIST, 0)
+for word in KEY_LIST:
+    if word in NUM_OF_WORDS:
+        Num_of_word = NUM_OF_WORDS.get(word)
+        Num_of_word += 1
```

Comment:
Is this naming meant to be so similar? One is a number (int), the other is a dictionary, but the names are almost identical, and neither of them follow the PEP8 standard (lower_case).